### PR TITLE
add python 3.X support

### LINKED ISSUE #1: Not working with latest python and slurm versions
I tried this, changing the code to work with python 3 by wrapping all print commands in () and adding the .decode('utf-8') to the subprocess.check_output command, but still am not seeing meaningful output when running under python 3.6.8 and slurm 2.11.03.

Let me know if I can provide more details but meanwhile wanted to see if you are still using and maintaining this tool Thanks!

### PR DESCRIPTION
**Pull request by matthdan**
Hi all, 

Feel free to update the code to python 3.6.X functional version ! 
Thank's for the work ! 

Matth

### CODE REVIEW

File: jobstats [hunk 1]
```diff
@@ -90,29 +91,29 @@ def job_state(USER,state,start=None,end=None):
...
+    count_jobs = len(subprocess.check_output(count_cmd, shell=True).splitlines())
+    print(out.decode())
+    print("Total Jobs: {}".format(count_jobs))
 
 
 def main():
     """Start of main program."""
 
-    userlist = subprocess.check_output("sacctmgr list user -n --parsable2 format=user",shell="True").splitlines()
+    userlist = subprocess.check_output("sacctmgr list user -n --parsable2 format=user", shell=True).splitlines()
```
Comment (shahzebsiddiqui) [inline jobstats]: I think there is an issue with output it returns in byte form so when i run `./jobstats -u $USER` it doesn't work i see an output list such as `b'zzhu1', b'zzq')` its a long list of user names and it doesn't match. Can you convert all user names into byte list.

**Comment by shahzebsiddiqui** [comment]
Hi @matthdan thanks for the contribution, i think there is one issue that needs to be addressed. It would be nice if we can add error check on python version at start of `main` to exit of python version doesn't match. You can use https://docs.python.org/3/library/platform.html#platform.python_version or any library that can check python version

File: jobstats (hunk 1 above)
Comment (matthdan) [reply jobstats]: Hello @shahzebsiddiqui, thank for your vigilance :+1: , I've corrected the comportment in the following commit 
3bb63f8.

Comment (shahzebsiddiqui) [reply jobstats]: thank you @mattdan i tested this locally and it worked out as I expected. the error is gone and it raise an error on python 2.x

**Review by shahzebsiddiqui — APPROVED** [review]

**Pull request merged**